indexers/schema.graphql

```diff
@@ -100,3 +203,37 @@ type Transfer @entity @compositeIndexes(fields: [["from", "to"]]) {
     """
     to: String! @index
 }
+
+"""
+Supported Event Types
+"""
+enum EventType {
+    EVM_HOST_POST_REQUEST
+    EVM_HOST_POST_REQUEST_HANDLED
+    EVM_HOST_POST_RESPONSE
+    EVM_HOST_POST_RESPONSE_HANDLED
+    HANDLER_V1_STATE_MACHINE_UPDATED
```

Comment:
State machine updated is now on the EvmHost.

Reply:
The timeout handled events are still missing

Reply:
The timeout handled events have been added, and state machine updated moved as well